addons/medical_gui/functions/fnc_updateInjuryList.sqf

```diff
@@ -31,7 +31,26 @@ private _bodyPartName = [
     LSTRING(RightLeg)
 ] select _selectionN;
 
-_entries pushBack [localize _bodyPartName, [1, 1, 1, 1]];
+private _bodyPartDamage = (_target getVariable [QEGVAR(medical,bodyPartDamage), [0, 0, 0, 0, 0, 0]]) select _selectionN;
+private _damageThreshold = GET_DAMAGE_THRESHOLD(_target);
+
+switch (true) do {
+    case (_selectionN > 3): {
+        _damageThreshold = LIMPING_DAMAGE_THRESHOLD * 4;
+    };
+    case (_selectionN > 1): {
+        _damageThreshold = FRACTURE_DAMAGE_THRESHOLD * 4;
+    };
+    case (_selectionN == 0): {
+        _damageThreshold = _damageThreshold / 2;
+    };
```

Comment:
IMO each `case` could use a comment to which body parts it applies, easier to follow.